src/web/editor.template.html

```diff
@@ -73,6 +73,7 @@
 </div>
 </div>
 <div id="footer"></div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js"></script>
```

Comment:
What breaks if we don't include this?  CPO/Pyret assumes an ES5 browser, so I don't want to add extra dependencies if it's avoidable.